docker/Dockerfile

```diff
@@ -94,16 +94,21 @@ ARG PYTHON_VER
 ################################ Stage: base (intermediate build target; basis for all other images herein)
 
 FROM python:${PYTHON_VER}-slim as base
+ARG PYTHON_VER
```

Comment:
`TODO` remove. Long story here: `nodejs` apt package requires `python3-minimal` which sadly there don't seem to be a way to get it to use our baked-in Python. Something to refine later.